src/backend/env/proposals.rs

```diff
@@ -93,11 +99,17 @@ impl Proposal {
                 }
             }
             Payload::Rewards(Rewards {
-                votes, receiver, ..
+                submissions,
+                receiver,
+                ..
             }) => {
                 if receiver == &principal {
                     return Err("reward receivers can not vote".into());
                 }
+                let user_id = state
```

Comment:
Nit: we have `user` available and could use `user.id`.

Reply:
Great, thanks so much!